mwdblib/object.py

```diff
@@ -161,6 +161,20 @@ def shares(self) -> List["MWDBShare"]:
             )
         return [MWDBShare(self.api, share, self) for share in self.data["shares"]]
 
+    @property
+    def attributes(self) -> Dict[Any, List[Any]]:
```

Comment:
I think it's JSON, so:

```suggestion
    def attributes(self) -> Dict[str, List[Any]]:
```

Reply:
✔️ I agree, attribute keys are strings.